components/Spotify/SpotifyPlayerModule.js

```diff
@@ -0,0 +1,47 @@
+import { useAtom, atom } from "jotai";
+import { useSession } from "next-auth/react";
+
+export const initialDeviceID = atom("");
+
+export default function SpotifyPlayerModule({ initialPlayerModuleRef }) {
```

Comment:
it might be worth thinking about merging this component with the `SpotifyPlayer` or making the rendering component the default export of `index.js`, since this component itself does not render anything.

Reply:
I tried to have everything related to the SDK on one page at the beginning but somehow ended up with the extra component. I don't remember why, to be honest. I will try to refactor the `PlayerModule` back into the `Player` at some later point.

Reply:
Just now I tried to change it, but when I try to put the `window.onSpotifyWebPlaybackSDKReady = () => ....` part in the function part of the `SpotifyPlayer.js`, I get "window is not defined".

Putting it into a component was a good way for me to ensure that it was mounted before the User can press any player buttons if that makes any sense... 
However, I am very open to other working solutions.